bpf/include/vmlinux.h

```diff
@@ -1,5 +1,6 @@
-#ifndef __VMLINUX_
-#define __VMLINUX_
+/* SPDX-License-Identifier: (LGPL-2.1 OR BSD-2-Clause) */
```

Comment:
could you please move all these header license changes into separate commit?

Reply:
sure :)